pkg/testutil/fakes.go

```diff
@@ -782,3 +807,586 @@ func (s *FakeSecrets) Delete(ns, name string) error {
+		namespace:       namespace,
+		cohOprClientSet: s.cohOprClientSet,
+	}
+}
+
+// simpleCohClusterNamespaceLister implements the CohClusterNamespaceLister interface.
+type simpleCohClusterNamespaceLister struct {
+	namespace       string
+	cohOprClientSet cohoprclientset.Interface
+}
+
+// List lists all CohClusters for a given namespace.
+func (s simpleCohClusterNamespaceLister) List(selector labels.Selector) (ret []*cohv1beta1.CohCluster, err error) {
+	var clusters []*cohv1beta1.CohCluster
+
+	list, err := s.cohOprClientSet.VerrazzanoV1beta1().CohClusters(s.namespace).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	for i := range list.Items {
+		cluster := list.Items[i]
+		if selector.Matches(labels.Set(cluster.Labels)) {
+			clusters = append(clusters, &cluster)
+		}
+	}
+	return clusters, nil
+}
+
+// Get retrieves the CohCluster for a given namespace and name.
+func (s simpleCohClusterNamespaceLister) Get(name string) (*cohv1beta1.CohCluster, error) {
+	return s.cohOprClientSet.VerrazzanoV1beta1().CohClusters(s.namespace).Get(context.TODO(), name, metav1.GetOptions{})
+}
+
+// ----- simpleCoherenceClusterLister
+// simpleCoherenceClusterLister implements the CoherenceClusterLister interface.
+type simpleCoherenceClusterLister struct {
+	kubeClient          kubernetes.Interface
+	cohClusterClientSet cohcluclientset.Interface
+}
+
+// List lists all CoherenceClusters.
+func (s *simpleCoherenceClusterLister) List(selector labels.Selector) (ret []*cohv1.CoherenceCluster, err error) {
+	namespaces, err := s.kubeClient.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	var clusters []*cohv1.CoherenceCluster
+	for _, namespace := range namespaces.Items {
+
+		list, err := s.CoherenceClusters(namespace.Name).List(selector)
+		if err != nil {
+			return nil, err
+		}
+		clusters = append(clusters, list...)
+	}
+	return clusters, nil
+}
+
+// CoherenceClusters returns an object that can list and get CoherenceClusters.
+func (s *simpleCoherenceClusterLister) CoherenceClusters(namespace string) cohclulister.CoherenceClusterNamespaceLister {
+	return simpleCoherenceClusterNamespaceLister{
+		namespace:       namespace,
+		cohOprClientSet: s.cohClusterClientSet,
+	}
+}
+
+// coherenceClusterNamespaceLister implements the CoherenceClusterNamespaceLister interface.
+type simpleCoherenceClusterNamespaceLister struct {
+	namespace       string
+	cohOprClientSet cohcluclientset.Interface
+}
+
+// List lists all CoherenceClusters for a given namespace.
+func (s simpleCoherenceClusterNamespaceLister) List(selector labels.Selector) (ret []*cohv1.CoherenceCluster, err error) {
+	var clusters []*cohv1.CoherenceCluster
+
+	list, err := s.cohOprClientSet.CoherenceV1().CoherenceClusters(s.namespace).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	for i := range list.Items {
+		cluster := list.Items[i]
```

Comment:
One comment (not saying this has to be changed).  We are getting a deep copy of cluster then just appending the address.  We can avoid the copy and eliminate cluster var, calling the list clusterList for clarity.
		if selector.Matches(labels.Set(clusterList.Items[i].Labels)) {
			clusters = append(clusters, clusterList.Items[i])

This same pattern of copying then using the address is used in several places.
		}

Reply:
okay, good point.  I guess it would have to be a pointer on this line ... 
```
			clusters = append(clusters, &clusterList.Items[i])
```
since 
```
clusters []*cohv1.CoherenceCluster
```

Reply:
Thanks, clean up this and other places.